### DISCUSSION TITLE
v0.63 does wonders!

### DISCUSSION THREAD

**Discussion by bel57** [seerge/g-helper#408] [Show and tell]
Heya, sorry I didn't know exactly where to post this. ~~~USER FEEDBACK~~~
I've read the release notes for 0.63, still not sure what kicked in but it's a solid improvement.
My G14 (2020) has a tendency to overheat but since I installed G-Helper it has helped a lot, and with each version it's getting better. 

I'm glad to report that while running v0.63 my laptop is super silent and temp excellent. 

Not a bug, not a feature request.. just kudos to the dev and users contributing towards all the good work, keep it up! 🙂

**Comment by seerge** [comment]
@bel57 hello, glad it works ok. I have moved this topic to discussions as this is not an issue.

**Comment by bel57** [comment]
Oooh sorry, I didn't even notice that tab to be honest. Thank you!!
Also forgot to mention I'm looking forward to the autoupdater.

**Reply by seerge** [reply]
@bel57 autoupdated is live in 0.63, there is no new versions to update to

**Reply by bel57** [reply]
Dope! In #364 nobody is talking about an embedded changelog. Sorry if my question is dumb but did you add it with the autoupdater v0.63? I'm aware the build is underlined and clicking on it opens a page with said changelog.
Idk if it would be possible to add it in the app.

**Reply by seerge** [reply]
Yes, as i just told above :) 0.63 will offer to download and install a new version on start of the app (if it's available)
